examples/internal/proto/examplepb/unannotated_echo_service.proto

```diff
@@ -13,12 +13,29 @@ package grpc.gateway.examples.internal.examplepb;
 // Do not need annotations.proto, can still use well known types as usual
 import "google/protobuf/duration.proto";
 
+// Embedded represents a message embedded in SimpleMessage.
+message UnannotatedEmbedded {
+	oneof mark {
+		int64 progress = 1;
+		string note = 2;
+	}
+}
+
 // UnannotatedSimpleMessage represents a simple message sent to the unannotated Echo service.
 message UnannotatedSimpleMessage {
 	// Id represents the message identifier.
 	string id = 1;
 	int64 num = 2;
 	google.protobuf.Duration duration = 3;
+	oneof code {
+		int64 line_num = 4;
+		string lang = 5;
+	}
+	UnannotatedEmbedded status = 6;
+	oneof ext {
+		int64 en = 7;
+		UnannotatedEmbedded no = 8;
+	}
```

Comment:
Why did you change the messages?

Reply:
Integration tests works with EchoService so I made UnnnnotatedMessage more similar to EchoMesage to reuse all tests.